src/v3/src/transformer/phone/phoneEnrollmentCodeTransformer.ts

```diff
@@ -28,14 +28,17 @@ export const transformPhoneCodeEnrollment: IdxStepTransformer = ({ transaction,
 
   let reminderElement: ReminderElement | undefined;
 
+  const { methods, profile } = nextStep.relatesTo?.value || {};
+  const { phoneNumber } = profile || {};
+  const methodType = methods?.[0]?.type;
   const resendStep = availableSteps?.find(({ name }) => name?.endsWith('resend'));
   if (resendStep) {
     const { name } = resendStep;
     reminderElement = {
       type: 'Reminder',
       options: {
-        content: loc('oie.phone.verify.sms.resendText', 'login'),
-        buttonText: loc('email.button.resend', 'login'),
+        content: methodType === 'sms' ? loc('oie.phone.verify.sms.resendText', 'login') : loc('oie.phone.verify.call.resendText', 'login'),
+        buttonText: methodType === 'sms' ? loc('oie.phone.verify.sms.resendLinkText', 'login') : loc('oie.phone.verify.call.resendLinkText', 'login'),
```

Comment:
consider pulling `methodType === 'sms'` out into a variable or even moving it up and renaming `methodType` to contain the boolean value of this check